nemo/collections/asr/parts/submodules/jasper.py

```diff
@@ -375,7 +375,7 @@ def update_masked_length(self, max_len, seq_range=None, device=None):
             self.lens = self.lens.to(device)
         else:
             self.lens = seq_range
-            self.max_len = max_len
+            self.max_len = torch.tensor(max_len)
```

Comment:
Does `max_len` need to be torch.int64 or long?

Reply:
I am not sure, but I assume (and tested) that this works because the other parts of the code do the same thing without specifying the dtype.

https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/asr/parts/submodules/jasper.py#L321
https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/asr/parts/submodules/jasper.py#L231

Reply:
then it looks good to me.